### PR TITLE
APPS-3505 Added test helper creation to AddCrudFacade spryk

### PR DESCRIPTION
**Pull request by miroshnikov-spryker**

### CODE REVIEW

File: config/spryk/spryks/Spryker/Zed/Business/AddCrudFacade.yml [hunk 1]
```diff
@@ -211,3 +211,5 @@ postSpryks:
                       value: "is_active"
                   propertyType:
                       value: "BOOLEAN"
+    # Add test helper
+    -   AddDomainEntityTestHelper
```
Comment (stereomon) [inline config/spryk/spryks/Spryker/Zed/Business/AddCrudFacade.yml:215]:
```suggestion
    - AddDomainEntityTestHelper
```

File: config/spryk/spryks/Spryker/Common/Test/EnableTestHelper.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    afterElement:
+        isOptional: true
+
+    addToElement:
+        value: "suites.{{ layer }}.modules.enabled"
+
+    targetPath:
+        value: "tests/{{ organization }}Test/Zed/{{ module }}/codeception.yml"
+
+    content:
```
Comment (stereomon) [inline config/spryk/spryks/Spryker/Common/Test/EnableTestHelper.yml:27]: Why this?

Comment (miroshnikov-spryker) [reply config/spryk/spryks/Spryker/Common/Test/EnableTestHelper.yml:27]: To make helperClassName an alias to the content parameter

Comment (stereomon) [reply config/spryk/spryks/Spryker/Common/Test/EnableTestHelper.yml:27]: Ok

**Review by stereomon — APPROVED** [review]

**Pull request merged**